screens/Relations/Relations.js

```diff
@@ -31,30 +31,53 @@ class RelationsComponent extends Component {
         super(props);
         this.state = {
             arrange_by: undefined,
+            is_ascending: true,
         };
     }
+
     onArrangeChange(value: string) {
         this.setState({
             arrange_by: value
         });
     }
 
+
+
+    populateRelations = () => {
+        return _.map(this.props.all_relations, (item, index)=>{
+            const toDetails = NavigationActions.navigate({
+                routeName: 'Details',
+              
+                params: {},
```

Comment:
We need to add a param here, probably the item's id so that we can populate the details screen.

Reply:
Follow the article
https://medium.com/@shovonroy/how-to-properly-navigate-with-react-navigation-e38fe3bf7381
and it should be an easy fix.